scalastyle-config.xml

```diff
@@ -87,7 +87,7 @@
     </check>
     <check level="warning" class="org.scalastyle.scalariform.CyclomaticComplexityChecker" enabled="true">
         <parameters>
-            <parameter name="maximum"><![CDATA[30]]></parameter>
+            <parameter name="maximum"><![CDATA[35]]></parameter>
```

Comment:
Even 30 is quite hight TBH....

Reply:
Yeah it is but it's just the nature of the transformer - might be worth splitting out all the code into seperate methods and the core `toEtmp/fromEtmp` is just a simple for-comp